### ISSUE TITLE
LEM1802 colors inverted

### ISSUE THREAD

**Issue by iversc** [HerobrinesArmy/DevCPU#47]
The foreground and background colors for text in the LEM1802 are inverted.   According to the spec, the highest four bits in each word are the foreground color, and the next four bits are the background color.  Assuming video memory is assigned to 0x1000, the line below:

SET [0x1000], 0x0F61

Should print a lowercase 'a' in white text on a black background.  Instead, it does the opposite.

**Comment by HerobrinesArmy** [comment]
I'm going to reply to this without actually looking at DevCPU or the spec. Someone (anyone) correct me if I write something wrong. I find it difficult to believe that the byte order for the tile color is wrong in DevCPU. I base that solely on the knowledge that I've run countless programs from all sorts of people in DevCPU and the background/foreground colors never appeared to be swapped. So either nearly everyone, including myself, misunderstood the LEM1802 spec, or DevCPU is compliant to the LEM1802 spec. Is it possible you misinterpreted the spec or the byte order of the words?

**Comment by iversc** [comment]
Thank you for taking the time to reply.  I brought this up because the results for DevCPU were not matching the results of a couple of other emulators I used.  I thought I had double-checked the spec at that point, and apparently, I got it backwards, and so did they.

I checked it again just now, and DevCPU has it correct.  I also verified against a different set of emulators.

Sorry for the time waste!  (And off to go file some other bugs in other emulators...)
